### PR TITLE
Use namespaced version of PHPUnit's TestCase

### PR DESCRIPTION
**Pull request by MarioBlazek**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "psr/log": "^1.0.1"        
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8 || ^5.0",
+        "phpunit/phpunit": "^4.8.35 || ^5.7",
```
Comment (prisis) [inline composer.json]: No need for the old 4.8 version, use only the new one

**Comment by SvenRtbg** [comment]
How would you then test against the lowest supported PHP 5.5.9?
Mit freundlichen Grüßen

Sven Rautenberg

**Comment by MarioBlazek** [comment]
@prisis PHPUnit 4.8 is required for PHP 5.5, PHPUnit v5 has minimum requirement of 5.6

**Comment by prisis** [comment]
Forgot that we still support PHP 5.5 and 5.6...

**Comment by MarioBlazek** [comment]
@prisis can this one be merged ?

**Comment by prisis** [comment]
looks good to me, but i cant merge it, sorry

@staabm, @denis-sokolov

**Comment by denis-sokolov** [comment]
Thank you for your contribution!

**Comment by MarioBlazek** [comment]
@denis-sokolov thank you for merge.

**Pull request merged**